sbi/neural_nets/flow.py

```diff
@@ -3,16 +3,18 @@
 
 
 from functools import partial
-from typing import Optional
+from typing import Optional, Sequence, Union
```

Comment:
the module structure is a bit confusing now I think: this `flow.py` file could be renamed to something like `flow_builders.py` or so? and moved to `utils`? not sure.. 

Also, `mdn.py` and `mnle.py` contain both "builders" and actual neural nets, and density estimators.

So I guess we should make an issue and a PR later that does some refactoring of the `neural_nets` module.

Reply:
Create Issue: Refactor neural_nets module structure.